app/controllers/projects_controller.rb

```diff
@@ -5,15 +5,15 @@ class ProjectsController < ApplicationController
 
   def index
     authorize Project
-    @pagy, @projects = pagy(policy_scope(Project.includes(:task_list).open))
+    @pagy, @projects = pagy(policy_scope(Project.open))
```

Comment:
The changes in this file belong to the next commit, I think, as it's removing the formatted conversion date (`converting_on_date` method) which triggered the Bullet error for this.